modules/mitre_software/mmd.json

```diff
@@ -771,7 +756,16 @@
     ],
     "system": false,
     "module": "mitre_software",
-    "uuid": "2a9663a6-a7bc-4716-aacf-ffcfad165846",
+    "softDeleteable": false,
+    "archivable": false,
+    "paused": false,
+    "partitionBy": "modifyDate",
+    "archivalCriteria": {
+        "keep_primary_data_for": 24,
```

Comment:
Do we need this?

Reply:
Yes, as we are importing complete module

Reply:
Yes, as we importing the complete module.